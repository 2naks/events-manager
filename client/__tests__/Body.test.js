import Body from '../components/Body.jsx';

describe('Body of Landing page should render self', () => {
  it('it should render for body', () => {
    const { wrapper } = shallow(<Body />);
    expect(wrapper.length).toEqual(1);
  });
});
