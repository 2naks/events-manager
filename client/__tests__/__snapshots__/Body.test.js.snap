// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body of Landing page should render self it should render for body 1`] = `undefined`;
